extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/internal/ValueSet.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.dmx.internal;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.eclipse.smarthome.core.library.types.PercentType;
+
+/**
+ * The {@link ValueSet} holds a set of values and fade times
+ *
+ * @author Jan N. Klug - Initial contribution
+ */
+
+public class ValueSet {
+    protected static final Pattern VALUESET_PATTERN = Pattern.compile("^(\\d*):([\\d,]*):([\\d-]*)$");
+
+    private int fadeTime;
+    private int holdTime;
+
+    private List<Integer> values = new ArrayList<Integer>();
+
+    /**
+     * constructor with fade times only
+     *
+     * @param fadeTime
+     *            fade time in ms
+     * @param holdTime
+     *            hold time in ms, -1 is forever
+     */
+    public ValueSet(int fadeTime, int holdTime) {
+        this.fadeTime = fadeTime;
+        this.holdTime = holdTime;
+    }
+
+    /**
+     * constructor with fade times and value
+     *
+     * @param fadeTime
+     *            fade time in ms
+     * @param holdTime
+     *            hold time in ms, -1 is forever
+     * @param value
+     *            DMX value (0-255)
+     */
+    public ValueSet(int fadeTime, int holdTime, int value) {
+        this.fadeTime = fadeTime;
+        this.holdTime = holdTime;
+        addValue(value);
```

Comment:
Below you use `Util.toDmxValue(value)` to ensure that the value is within range. I think you should do that here as well.